penaltymodel_maxgap/penaltymodel/maxgap/generation.py

```diff
@@ -57,6 +70,9 @@ def generate_ising(graph, feasible_configurations, decision_variables,
             to a non-zero infeasible gap.
 
     """
+    if len(graph) == 0:
+        return dimod.BinaryQuadraticModel.empty(dimod.SPIN), float('inf')
```

Comment:
Good idea. What do you think makes more sense?

Reply:
I like a gap of infinity because I think it's unlikely that a user would choose that as a gap. Hence, when you see it's an infinity you know it's an error or a computer generated default.

Reply:
I think this is out-of-scope for this PR then. See #79